experiments/ClimaEarth/user_io/diagnostics_plots.jl

```diff
@@ -107,6 +107,11 @@ function make_plots_generic(
     grid_pos = 1
 
     for var in vars
+        if all(isnan.(var.data))
```

Comment:
You can also write this
```suggestion
        if all(isnan, var.data)
```